testing/cloud_layouts/script.d/wait_cluster_ready/info_script.sh

```diff
@@ -20,7 +20,7 @@ kubectl -n d8-system get deploy/deckhouse -o wide
 echo "Pod/deckhouse-*"
 kubectl -n d8-system get po -o wide | grep ^deckhouse
 echo "Enabled modules:"
-kubectl -n d8-system exec deploy/deckhouse -- deckhouse-controller module list -o yaml | grep -v enabledModules: | sort
+kubectl -n d8-system exec svc/deckhouse-leader -- deckhouse-controller module list -o yaml | grep -v enabledModules: | sort
```

Comment:
I'm not sure if changes are required here either, if necessary I'll delete it

Reply:
Let's keep them. Doesn't look harmful